main.tf

```diff
@@ -6,9 +6,12 @@ module "codepipeline_label" {
   context = module.this.context
 }
 
-resource "aws_s3_bucket" "default" {
+module "artifact_store_bucket" {
```

Comment:
Hmmm this is a bit of scope creep. I don't think we want to use the s3 bucket module in this PR. Let's keep that separated.